apps/openmw/mwclass/activator.cpp

```diff
@@ -33,8 +33,7 @@ namespace MWClass
     {
         if (!model.empty())
         {
-            renderingInterface.getObjects().insertModel(ptr, model, true);
-            ptr.getRefData().getBaseNode()->setNodeMask(MWRender::Mask_Static);
+            renderingInterface.getObjects().insertModel(ptr, model, MWRender::Mask_Static, true);
```

Comment:
@akortunov masks do stay the same insertModel set it to Object and it's hacked at exit settings it static...